internal/clifs/clifs.go

```diff
@@ -5,85 +5,159 @@ import (
 	"fmt"
 	"io/fs"
 	"os"
+	"path"
 	"strings"
 
 	"github.com/pkg/errors"
 	"github.com/tkhq/tkcli/internal/apikey"
 )
 
-// Given a user-specified directory, return the path.
-// The logic is in the case where users do not specify a folder.
-// If the folder isn't specified, we default to $XDG_CONFIG_HOME/.config/turnkey/keys.
-// If this env var isn't set, we default to $HOME/.config/turnkey/keys
-// If $HOME isn't set, this function returns an error.
-func GetKeyDirPath(userSpecifiedPath string) (string, error) {
-	if userSpecifiedPath == "" {
-		var configHome string
-		if os.Getenv("XDG_CONFIG_HOME") != "" {
-			configHome = os.Getenv("XDG_CONFIG_HOME")
-		} else {
-			homeDir, err := os.UserHomeDir()
-			if err != nil {
-				return "", errors.Wrap(err, "error while reading user home directory")
-			}
-			configHome = homeDir + "/.config"
-		}
-		return configHome + "/turnkey/keys", nil
-	} else {
-		if _, err := os.Stat(userSpecifiedPath); !os.IsNotExist(err) {
-			return userSpecifiedPath, nil
+var keysDirectory string
+
+const (
+	// DefaultKeyName is the name of the default API key
+	DefaultKeyName = "default"
```

Comment:
Is there a need for a default? I'd rather prompt the user for a name in 100% of the cases

Reply:
Even from the end user perspective?  Do you anticipate the average user ever having more than one API key?  Why must they assert a key name every time?

Reply:
I guess that's true. The average user (or machine) will have a single Turnkey API key. Considered my mind changed, let's keep the default behavior 👍